src/LightStep/Options.cs

```diff
@@ -115,6 +120,13 @@ public Options WithMaxBufferedSpans(int count)
             ReportMaxSpans = count;
             return this;
         }
+
+        public Options WithJsonReports(bool shouldUseJson)
```

Comment:
Suggestion: instead of having an option per transport alternative, it may be better to have a single option that takes an enum value. This way, if/when a third option (such as gRPC) is added, it will still only be possible to select a single option.

Reply:
Good idea.